<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDEM-ARCHIVER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>541</width>
      <height>250</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Encoder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Dictionary</string>
         </property>
         <widget class="QRadioButton" name="naive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>NAIVE</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="bm">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Boyer-Moore</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Threads</string>
         </property>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>47</width>
            <height>24</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>521</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Input File Path</string>
         </property>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>20</y>
            <width>90</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>20</y>
            <width>390</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Destination Folder</string>
         </property>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>20</y>
            <width>390</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>20</y>
            <width>90</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>521</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Input File Path</string>
         </property>
         <widget class="QPushButton" name="dec_inputButton">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>20</y>
            <width>90</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="dec_inputPath">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>20</y>
            <width>390</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Destination Folder</string>
         </property>
         <widget class="QTextBrowser" name="dec_destPath">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>20</y>
            <width>390</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="dec_destButton">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>20</y>
            <width>90</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>331</width>
        <height>107</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="dec_Threads">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Threads</string>
         </property>
         <widget class="QSpinBox" name="dec_threads">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>47</width>
            <height>24</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dec_startButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
